src/accountsMetadata.ts

```diff
@@ -0,0 +1,106 @@
+import { OutputVendorName } from '@/originalBudgetTrackingApp/configManager/configManager';
+import { CompanyTypes } from 'israeli-bank-scrapers-core';
+
+type AccountKey = CompanyTypes | OutputVendorName;
+
+export type AccountMetadata = {
+  companyKey: string;
+  companyName: string;
+  logo: string;
+}
+
+const PUBLIC_STORAGE_PREFIX = 'https://budget-tracking-public.s3.eu-central-1.amazonaws.com/logos';
+
+const ACCOUNT_METADATA: Record<AccountKey, AccountMetadata> = {
+  [CompanyTypes.max]: {
+    companyKey: CompanyTypes.max,
+    companyName: 'Max',
+    logo: buildLogoUrl('max.jpeg',)
+  },
+  [CompanyTypes.amex]: {
+    companyKey: CompanyTypes.amex,
+    companyName: 'American Express',
+    logo: buildLogoUrl('americanExpress.jpeg')
+  },
+  [CompanyTypes.beinleumi]: {
+    companyKey: CompanyTypes.beinleumi,
+    companyName: 'הבינלאומי',
```

Comment:
I think we need a font for Hebrew... just note

Reply:
ok. Lets deal with that when we do some more investing in hebrew